### PR TITLE
Fix api doc issues

### PR DESCRIPTION
**Pull request by pkrull-ansys**

### CODE REVIEW

**Review by klmcadams — COMMENTED** [review]
This looks good to me! I would wait until @PipKat can do her review before merging this

File: doc/source/contributing.rst
Comment (PipKat) [inline doc/source/contributing.rst:9]:
```suggestion
in the *PyAnsys developer's guide*. Ensure that you are thoroughly familiar
```
Recent decision to follow Google style guidelines for using sentence case titles.

File: src/ansys/additive/core/additive.py
Comment (PipKat) [inline src/ansys/additive/core/additive.py:67]:
```suggestion
        gRPC channel connection to use for communicating with the server. If the
        default ``None`` is used, a connection is established using the host and port
        parameters. This parameter is ignored if the ``server_channels`` parameter
        is other than ``None``.
```
After the brief description, use complete sentences and avoid future tense. In the last
line, did you mean ``server_channels``? I just don't see this parameter and wanted to
be sure it was what you wanted.

File: src/ansys/additive/core/additive.py
Comment (PipKat) [inline src/ansys/additive/core/additive.py:70]:
```suggestion
        Host name or IPv4 address of the server. This parameter is ignored if the ``server_channels`` or ``channel``
        parameters is other than ``None``.
```

File: src/ansys/additive/core/additive.py
Comment (PipKat) [inline src/ansys/additive/core/additive.py:196]:
```suggestion
            list is returned.
```

File: src/ansys/additive/core/additive.py
Comment (PipKat) [inline src/ansys/additive/core/additive.py:352]: I assume "Material Tuning Tool" is initial caps in the documentation? Today, technical documentation avoids unnecessary capitalization, reserving it for proper nouns as it should! However, legacy Ansys product documentation uses initial capital letters everywhere--so I get it if you are matching the name (capitalization) in the existing documentation.

File: src/ansys/additive/core/geometry_file.py
Comment (PipKat) [inline src/ansys/additive/core/geometry_file.py:67]:
```suggestion
        """Initialize a ``BuildFile`` object."""
```

File: src/ansys/additive/core/microstructure.py
Comment (PipKat) [inline src/ansys/additive/core/microstructure.py:46]: Here "m`` is used instead of "meters"--you might want to be consistent.

File: src/ansys/additive/core/microstructure.py
Comment (PipKat) [inline src/ansys/additive/core/microstructure.py:64]:
```suggestion
    """Minimum cushion between sensor dimension and sample size in the X and Y dimensions (m)."""
```
For coordinates, you used lower case x, y, z. Here, you are using upper case. Just pointing this out....

File: src/ansys/additive/core/microstructure.py
Comment (PipKat) [inline src/ansys/additive/core/microstructure.py:66]:
```suggestion
    """Minimum cushion between sensor dimension and sample size in the Z dimension (m)."""
```

File: src/ansys/additive/core/microstructure.py
Comment (PipKat) [inline src/ansys/additive/core/microstructure.py:94]:
```suggestion
    """The default random seed, which indicates that a random seed was not provided."""
```
I'm not sure I fully understand this sentence. Could it be rewritten as: The default random seed, which is used if a random seed is not provided.

File: src/ansys/additive/core/microstructure.py
Comment (PipKat) [inline src/ansys/additive/core/microstructure.py:282]:
```suggestion
        When setting the```sample_size_x`` parameter, the value must be greater than the ``sensor_dimension``
        value plus the :obj:`MIN_XY_SIZE_CUSHION` value.
```

File: src/ansys/additive/core/microstructure.py
Comment (PipKat) [inline src/ansys/additive/core/microstructure.py:300]:
```suggestion
        When setting the ``sample_size_y`` parameter, the value must be greater than the
        ``sensor_dimension`` value plus the :obj:`MIN_XY_SIZE_CUSHION` value.
```

File: src/ansys/additive/core/microstructure.py
Comment (PipKat) [inline src/ansys/additive/core/microstructure.py:318]:
```suggestion
        When setting the ``sample_size_y`` parameter, the value must be greater than the
        ``sensor_dimension`` value plus the :obj:`MIN_XY_SIZE_CUSHION` value.
```

File: src/ansys/additive/core/microstructure.py
Comment (PipKat) [inline src/ansys/additive/core/microstructure.py:235]:
```suggestion
        Valid values are from the :obj:`MIN_POSITION_COORDINATE` value to
        the :obj:`MAX_POSITION_COORDINATE` value.
```
If I missed earlier formations of this, please revise. For clarity, a noun should follow a code entity.

File: src/ansys/additive/core/microstructure.py
Comment (PipKat) [inline src/ansys/additive/core/microstructure.py:250]:
```suggestion
        Valid values are from the :obj:`MIN_POSITION_COORDINATE` value to the
        :obj:`MAX_POSITION_COORDINATE` value.
```

File: src/ansys/additive/core/microstructure.py
Comment (PipKat) [inline src/ansys/additive/core/microstructure.py:265]:
```suggestion
        Valid values are from the :obj:`MIN_POSITION_COORDINATE` value to the
         :obj:`MAX_POSITION_COORDINATE` value.
```

File: src/ansys/additive/core/microstructure.py
Comment (PipKat) [inline src/ansys/additive/core/microstructure.py:280]:
```suggestion
        Valid values are from the :obj:`MIN_SAMPLE_SIZE` value to the
        :obj:`MAX_SAMPLE_SIZE` value.
```

File: src/ansys/additive/core/microstructure.py
Comment (PipKat) [inline src/ansys/additive/core/microstructure.py:298]:
```suggestion
        Valid values are from the :obj:`MIN_SAMPLE_SIZE` value to the
        :obj:`MAX_SAMPLE_SIZE` value.
```

File: src/ansys/additive/core/microstructure.py
Comment (PipKat) [inline src/ansys/additive/core/microstructure.py:316]:
```suggestion
        Valid values are from the :obj:`MIN_SAMPLE_SIZE` value to the
        :obj:`MAX_SAMPLE_SIZE` value.
```

File: src/ansys/additive/core/microstructure.py
Comment (PipKat) [inline src/ansys/additive/core/microstructure.py:332]:
```suggestion
        Valid values are from the :obj:`MIN_SENSOR_DIMENSION` value to the
        :obj:`MAX_SENSOR_DIMENSION` value.
```

File: src/ansys/additive/core/microstructure.py
Comment (PipKat) [inline src/ansys/additive/core/microstructure.py:384]:
```suggestion
        Valid values are from the :obj:`MIN_COOLING_RATE` value to the
        :obj:`MAX_COOLING_RATE` value.
```

File: src/ansys/additive/core/microstructure.py
Comment (PipKat) [inline src/ansys/additive/core/microstructure.py:397]:
```suggestion
        Valid values are from the :obj:`MIN_THERMAL_GRADIENT` value to the
        :obj:`MAX_THERMAL_GRADIENT` value.
```

File: src/ansys/additive/core/microstructure.py
Comment (PipKat) [inline src/ansys/additive/core/microstructure.py:414]:
```suggestion
        Valid values are from the :obj:`MIN_MELT_POOL_WIDTH` value to the
        :obj:`MAX_MELT_POOL_WIDTH` value.
```

File: src/ansys/additive/core/microstructure.py
Comment (PipKat) [inline src/ansys/additive/core/microstructure.py:431]:
```suggestion
        Valid values are from the :obj:`MIN_MELT_POOL_DEPTH` value to the
        :obj:`MAX_MELT_POOL_DEPTH` value.
```

File: src/ansys/additive/core/microstructure.py
Comment (PipKat) [inline src/ansys/additive/core/microstructure.py:446]:
```suggestion
        Valid values are from the :obj:`MIN_RANDOM_SEED` value to the
        :obj:`MAX_RANDOM_SEED` value.
```

File: src/ansys/additive/core/porosity.py
Comment (PipKat) [inline src/ansys/additive/core/porosity.py:115]:
```suggestion
        Valid values are from the :obj:`MIN_SAMPLE_SIZE` value to the
        :obj:`MAX_SAMPLE_SIZE` value.
```
Was ``meters`` after the second object a mistake?

File: src/ansys/additive/core/porosity.py
Comment (PipKat) [inline src/ansys/additive/core/porosity.py:128]:
```suggestion
        Valid values are from the :obj:`MIN_SAMPLE_SIZE` value to the
        :obj:`MAX_SAMPLE_SIZE` value.
```
Same question here about ``meters``.

File: src/ansys/additive/core/porosity.py
Comment (PipKat) [inline src/ansys/additive/core/porosity.py:141]:
```suggestion
        Valid values are from the :obj:`MIN_SAMPLE_SIZE` value to the
        :obj:`MAX_SAMPLE_SIZE` value.
```
Same question here about meters.

File: src/ansys/additive/core/single_bead.py
Comment (PipKat) [inline src/ansys/additive/core/single_bead.py:33]:
```suggestion
    """Provides input parameters for a single bead simulation."""
```

File: src/ansys/additive/core/single_bead.py
Comment (PipKat) [inline src/ansys/additive/core/single_bead.py:109]:
```suggestion
        Valid values are from the :obj:`MIN_BEAD_LENGTH` value to the :obj:`MAX_BEAD_LENGTH` value.
```
Same question here about meters.

File: src/ansys/additive/core/thermal_history.py
Comment (PipKat) [inline src/ansys/additive/core/thermal_history.py:72]:
```suggestion
        """Transform this object into a ``RangeMessage`` object to send to the server."""
```

File: src/ansys/additive/core/thermal_history.py
Comment (PipKat) [inline src/ansys/additive/core/thermal_history.py:80]:
```suggestion
    """Minimum radius for the circular field of view of the sensor (m)."""
```
Here, and in the next few instances, are examples where adding the article really improves readability!

File: src/ansys/additive/core/thermal_history.py
Comment (PipKat) [inline src/ansys/additive/core/thermal_history.py:82]:
```suggestion
    """Maximum radius for the circular field of view of the sensor (m)."""
```

File: src/ansys/additive/core/thermal_history.py
Comment (PipKat) [inline src/ansys/additive/core/thermal_history.py:106]:
```suggestion
        Valid values are from the :obj:`MIN_SENSOR_RADIUS` value to the
        :obj:`MAX_SENSOR_RADIUS` value.
```

File: src/ansys/additive/core/thermal_history.py
Comment (PipKat) [inline src/ansys/additive/core/thermal_history.py:147]:
```suggestion
        """Initialize a ``ThermalHistoryInput`` object."""
```

File: src/ansys/additive/core/thermal_history.py
Comment (PipKat) [inline src/ansys/additive/core/thermal_history.py:207]:
```suggestion
            raise TypeError("Geometry must be an StlFile of BuildFile.")
```
Messages should conclude with punctuation. Can the message be changed to use the extension or name (given that I don't think Build is an extension but generic type): 
"Geometry must be an STL file or build file."

File: src/ansys/additive/core/thermal_history.py
Comment (PipKat) [inline src/ansys/additive/core/thermal_history.py:218]:
```suggestion
            raise TypeError("Coaxial average sensor inputs must be 'CoaxialAverageSensorInputs' objects.")
```
The original has an "a" --so maybe all inputs are provided in one object? I find this message a tad unclear.

**Review by PipKat — APPROVED** [review]
@pkrull-ansys Looks good--just a bunch of nitpicks.

File: src/ansys/additive/core/additive.py
Comment (pkrull-ansys) [reply src/ansys/additive/core/additive.py:352]: Yes. The currently released docs (23.2) use initial caps.

File: src/ansys/additive/core/machine.py
Comment (pkrull-ansys) [reply src/ansys/additive/core/machine.py:34]: Thanks! For brevity's sake, I think I'd prefer abbreviations. Is it ok to say `in m` rather in `in meters`? Also, for the angles, do I keep it `in degrees` or `in deg`? For `degrees Celsius`, do I drop `degrees` and use `temperature in C`?

**Pull request merged**
